polaris/hub/settings.py

```diff
@@ -30,12 +30,12 @@ class PolarisHubSettings(BaseSettings):
 
     hub_url: HttpUrlString = "https://polarishub.io/"
     api_url: Optional[HttpUrlString] = None
-    authorize_url: HttpUrlString = "https://pure-whippet-77.clerk.accounts.dev/oauth/authorize"
+    authorize_url: HttpUrlString = "https://clerk.polarishub.io/oauth/authorize"
```

Comment:
Does this increase the risk of devs accidentally doing things in prod or is there some other gate in place elsewhere that will prevent accidents?

Reply:
That's a great point. I'll add support for using a `.env`file for local development, so it's easy to point to dev for us.